apps/test/vue/src/assets/global.css

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Red+Hat+Display:wght@300;400;500;700&family=Red+Hat+Text:ital,wght@0,400;0,500;0,700;1,300&display=swap');
```

Comment:
Btw. we don't provide any info in our documentation that somebody should include Red Hat Display font, right?

Reply:
Good point, we should do it, I will create ticket

Reply:
https://vsf.atlassian.net/browse/SFUI2-1097